pom.xml

```diff
@@ -141,6 +141,7 @@
   <modules>
     <module>cdap-common-unit-test</module>
     <module>cdap-api</module>
+    <module>cdap-app-templates</module>
```

Comment:
I think @sreevatsanraman added this so the package also builds templates. 
I will remove it. Sounds good @sreevatsanraman